__tests__/app.test.js

```diff
@@ -136,4 +136,46 @@ describe("/api/articles", () => {
         expect(articles).toBeSortedBy("created_at", { descending: true });
       });    
   }) 
-});
\ No newline at end of file
+});
+
+describe("/api/articles/:article_id/comments", () => {
+  test("GET 200: returns an array of comments for the given article_id with the specified properties", () => {
+    return request(app)
+      .get("/api/articles/9/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments.length).toBe(2)
+        const expected = {
+          votes: 20,
+          author: "icellusedkars",
+          body: "The owls are not what they seem.",
+          article_id: 9
+        };
+        comments.forEach((comment) => {
+          if(comment.comment_id === 17){
```

Comment:
Im not sure theres that much point in doing a forEach if we're not checking all the comments. I would probably just check the key value types within the forEach rather checking the exact comment :)